Homework8basic/reduce.js

```diff
@@ -0,0 +1,8 @@
+var numbers = [1, 2, 3, 4, 5,]
+
+var reverstNumbers = numbers.reduce((number,toatal, index, array) => {
+    return array.reverse(number);
```

Comment:
Hehehe 😄  you can use this even without reduce. Smart way of solving this but I would appreciate if you would solve this problem without array.reverse. It's an exercise to get you accustomed to the reduce function and how we can manipulate arrays with it.